File: ffmpeg/android-ffmpeg/src/main/java/nl/bravobit/ffmpeg/FileUtils.java
package nl.bravobit.ffmpeg;

import android.content.Context;

import java.io.File;

class FileUtils {
    private static final String FFMPEG_FILE_NAME = "lib..ffmpeg..so";
    private static final String FFPROBE_FILE_NAME = "lib..ffprobe..so";

    static File getFFmpeg(Context context) {
        File folder = new File(context.getApplicationInfo().nativeLibraryDir);
        return new File(folder, FFMPEG_FILE_NAME);
    }

    static File getFFprobe(Context context) {
        File folder = new File(context.getApplicationInfo().nativeLibraryDir);
        return new File(folder, FFPROBE_FILE_NAME);
    }
}

File: ffmpeg/android-ffmpeg/src/main/java/nl/bravobit/ffmpeg/FileUtils.kt
package nl.bravobit.ffmpeg

import android.content.Context
import java.io.File

internal object FileUtils {

    private const val FFMPEG_FILE_NAME = "lib..ffmpeg..so"
    private const val FFPROBE_FILE_NAME = "lib..ffprobe..so"

    @JvmStatic
    fun getFFmpeg(context: Context): File {
        val folder = File(context.applicationInfo.nativeLibraryDir)
        return File(folder, FFMPEG_FILE_NAME)
    }

    @JvmStatic
    fun getFFprobe(context: Context): File {
        val folder = File(context.applicationInfo.nativeLibraryDir)
        return File(folder, FFPROBE_FILE_NAME)
    }
}